Automattic/jetpack
# AI Chat: Make it work on WPCOM

This makes the AI Chat work on WPCOM Simple sites.
More context:
peGwbA-Be-p2

## Does this pull request change what data or activity we track or use?

NO

## Testing instructions:

This block should work with both Jetpack **and** WPCOM sites.

1. Get a Jetpack site, connect to WPCOM
2. Purchase / enable Jetpack Search
3. `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );`
4. Publish a few posts
5. Insert the block on a page, publish
6. Open the page on the frontend
7. Ask a question relating to your published posts
8. Get a WPCOM site, sandbox public-api AND site domain
9. `bin/jetpack-downloader test jetpack chat-with-site/wpcom`
10. Get Jetpack search product ( click upgrade jetpack search on /wp-admin/admin.php?page=jetpack-search )
4. Publish a few posts
5. Insert the block on a page, publish
6. Open the page on the frontend
7. Ask a question relating to your published posts
